LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoraMessageWrapper.cs

```diff
@@ -179,7 +179,7 @@ public LoRaMessageWrapper(LoRaPayload payload, LoRaMessageType type, byte[] phys
             LoRaPayloadMessage = payload;
             PktFwdPayload = new DownlinkPktFwdMessage(Convert.ToBase64String(LoRaPayloadMessage.GetByteMessage()), datr, rfch, freq, tmst);
             var jsonMsg = JsonConvert.SerializeObject(PktFwdPayload);
-            Logger.Log(ConversionHelper.ByteArrayToString(payload.GetLoRaMessage().DevEUI.ToArray()),$"{((MType)(payload.Mhdr.Span[0])).ToString()} {jsonMsg}", Logger.LoggingLevel.Full);
+            Logger.Log(ConversionHelper.ByteArrayToString(payload.DevAddr.Span.ToArray()),$"{((MType)(payload.Mhdr.Span[0])).ToString()} {jsonMsg}", Logger.LoggingLevel.Full);

```

Comment:
We log the devaddr because we don't have yet the deveui right?

Reply:
Yes, Having the DevEUI here would be possible, but need a substantial refactor of the current downstream pipe. I will add a Workitem and we could prioritize this accordingly

Reply:
Not a priority but it is not great to follow the log.